compiler/rustc_codegen_llvm/src/gotoc/cbmc/irep/to_irep.rs

```diff
@@ -292,6 +291,11 @@ impl ToIrep for ExprValue {
             ExprValue::UnOp { op, e } => {
                 Irep { id: op.to_irep_id(), sub: vec![e.to_irep(mm)], named_sub: btree_map![] }
             }
```

Comment:
```suggestion
            },
```

Reply:
I think this would be a rustfmt violation?